.azure-pipelines/check-dirty-version.yml

```diff
@@ -0,0 +1,16 @@
+steps:
+- script: |
+    . functions.sh
+    SONIC_VERSION=$(sonic_get_version)
+    echo "SONIC_VERSION=$SONIC_VERSION"
+    if [ "$SONIC_VERSION" == *dirty* ]; then
+      # Print the detail dirty info
+      git status --untracked-files=no -s --ignore-submodules
+
+      # Exit with error, if it is a PR build
```

Comment:
We can block official build if the issue has fixed in all release branches.

Reply:
It seems that this step will change in future.
Can we use a template in master branch?

Reply:
The check-dirty-version.yml is a template file.